src/System/IO/Resource/Linear/Internal.hs

```diff
@@ -98,13 +100,22 @@ instance Control.Monad RIO where
 
 -- files
 
--- Remark: Handle needs to be private otherwise `Data.Coerce.coerce` could wreak
```

Comment:
Is the comment no longer relevant/wrong now? If so, is there anything else we could use to document `Handle`?

Reply:
This part of the comment is still correct. But I'm not sure that it's awfully relevant, because it was meant to say “well, we could add this other function in the future” and this PR does just that.

Reply:
This was not a docstring, by the way, just an internal comment in the source.

Reply:
It's not relevant anymore. See https://github.com/tweag/linear-base/pull/428#issuecomment-1289170485.